lib/logging.dart

```diff
@@ -314,7 +314,7 @@ class Level implements Comparable<Level> {
   ];
 
   @override
-  bool operator ==(Object other) => other is Level && value == other.value;
+  bool operator ==(Object? other) => other is Level && value == other!.value;
```

Comment:
The `?` on `Object?` is happening due to https://github.com/dart-lang/sdk/issues/38339.  The `!` on `other!.value` is happening due to https://github.com/dart-lang/sdk/issues/38340.